lib/videojs-playlist-ui.js

```diff
@@ -198,20 +199,20 @@ videojs.PlaylistMenu = videojs.Component.extend({
     }
   },
   items: [],
-  update() {
+  update(settings) {
     // replace the playlist items being displayed, if necessary
     const playlist = this.player_.playlist();
     if (this.items.length !== playlist.length) {
       // if the menu is currently empty or the state is obviously out
       // of date, rebuild everything.
-      this.createPlaylist_();
+      this.createPlaylist_(settings);
```

Comment:
can you verify whether `this.options()` would have the same stuff available? given that this is a component, I think it would. In which case, we wouldn't need to pass it in explicitly.